src/main/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraService.scala

```diff
@@ -0,0 +1,75 @@
+
+package org.broadinstitute.dsde.agora.server.webservice
+
+import org.broadinstitute.dsde.agora.server.model.{AgoraEntityType, AgoraEntity, AgoraEntityProjection}
+import org.broadinstitute.dsde.agora.server.webservice.util.ServiceMessages
+import org.broadinstitute.dsde.agora.server.webservice.validation.{AgoraValidation, AgoraValidationRejection}
+import org.joda.time.DateTime
+import spray.routing.{HttpService, Route}
+
+trait AgoraService extends HttpService with PerRequestCreator with AgoraDirectives {
+  this: ServiceHandlerProps =>
+
+  private implicit val ec = actorRefFactory.dispatcher
```

Comment:
This is the execution context for the magnet pattern the vault uses for vault commons. It is needed for the agora directives to get username from the token. I can rename it to executionContext